src/SprykerEco/Zed/Payone/Business/PayoneBusinessFactory.php

```diff
@@ -350,13 +680,23 @@ protected function getGlossaryFacade()
      *
      * @return \SprykerEco\Zed\Payone\Business\Payment\MethodMapper\GenericPayment
      */
-    protected function createGenericPayment(Store $store)
+    public function createGenericPayment(Store $store)
```

Comment:
* Add a return typehins to methods in the factory.

Reply:
++